### DISCUSSION TITLE
Hacktoberfest 2021

### DISCUSSION THREAD

**Discussion by Thihup** [piranhacloud/piranha#1940] [Q&A] [answered]
Will this project accept contributions from Hacktoberfest 2021?
If so, it is needed to add the "hacktoberfest" topic to the repo.

**Comment by mnriem** [comment, marked as answer]
@Thihup I am OK with it as long as they acknowledge that Manorrock.com owns their contributions in their first PR and you are the one taking point in helping them.

**Reply by mnriem** [reply]
I have added hacktoberfest and hacktoberfest2021 as topics to the repo
